src/Console/Thumbnails/GenerateThumbnailsCommand.php

```diff
@@ -28,41 +35,37 @@ protected function configure(): void
             );
     }
 
-    protected function execute(InputInterface $input, OutputInterface $output): void
+    public function __construct(Thumbnails $thumbnails)
     {
-        // Get input values
-        $folderOption = $input->getOption('folder');
+        $this->finder = new Finder();
```

Comment:
I wouldn't move this. It is only used in one method. I'm not sure if the constructor is always called or if there is an optimisation on it. But if it would always be called also when not using this command creating a new instance of finder is a waste of resources

Reply:
Fixed by c40f720